### DISCUSSION TITLE
How to integrate razorpay payments gateway in fastapi

### DISCUSSION THREAD

**Discussion by alam-mahtab** [fastapi/fastapi#3050] [Questions] [answered]
Want to integrate payments gateway in my project using fastapi, consider me as a beginner or a noob coder.
If you guys ever worked on payments gateway kindly help me in this.

**Comment by rozeappletree** [comment]
If you want a template, I can create one ;)

**Reply by SagarDollin** [reply]
Could you please leave a link to that? If possible, can you document for better understanding? Thanks!

**Comment by tiangolo** [comment, marked as answer]
I didn't know about it, it's probably better to ask them (Razorpay), and they will have a strong motivation to help you develop and debug your use case, as in the end you'll end up paying them. :sweat_smile: 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
